src/components/LandingPage/index.js

```diff
@@ -11,88 +11,88 @@ import { ReactComponent as Container } from '../../assets/images/container.svg';
 import { ReactComponent as Orch } from '../../assets/images/orcherstration.svg';
 import { ReactComponent as Registry } from '../../assets/images/registry.svg';
 import { ReactComponent as Cloud } from '../../assets/images/multicloud.svg';
-import './LandingPage.css';
+import Styles from './LandingPage.module.css';
```

Comment:
uppercase gives it some kind of importance. please change to lowercase as in `styles`

Reply:
a simple `ctrl find` and `replace` can save the day

Reply:
@kawerewagaba this has been fixed.